### ISSUE TITLE
There is no warn or error when tempdir is full, but I got a empty file.

### ISSUE THREAD

**Issue by WhyLogin** [mk-j/PHP_XLSXWriter#211]
Could you please add an error message for it? It takes me 3 hours to find out the reason.

**Comment by finlaywang** [comment]
Yes, for me my tempdir is unwritable but there is no warning at all, and waste me two days to figure it out..

**Comment by G1K** [comment]
Same problem. Thank you for creating a ticket I at least spent less time ))

**Comment by mk-j** [comment]
Sure, we'll add something for this.
